app/src/main/java/com/teambeme/beme/detail/adapter/ReplyAdapter.kt

```diff
@@ -37,7 +37,7 @@ class ReplyAdapter(private val context: Context, private val viewModel: DetailVi
 
         fun onBind(data: ReplyParentData) {
             binding.replyParentData = data
-            if (data.dataChild[0].txt_id != "") {
+            if (data.dataChild.size == 0 || data.dataChild[0].txt_id != "") {
```

Comment:
이런건 판별해주는 함수를 따로 만들어주는게 좋을 것 같아. 한 함수에 너무 많은 책임을 가지고 있어
```suggestion
            fun isDataNull() = data.dataChild.size == 0 || data.dataChild[0].txt_id != ""
            
            if(isDataNull())
                /* Put Anything Here */
```